src/main/java/com/example/decafe/sprites/Waiter.java

```diff
@@ -0,0 +1,101 @@
+package com.example.decafe.sprites;
+
+import java.io.FileNotFoundException;
+
+import com.example.decafe.Player;
+import com.example.decafe.util.ImageUtil; 
+
+import javafx.scene.image.ImageView;
+
+public class Waiter {
+  public ImageView waiterImageView;
+
+  public Waiter(ImageView waiterImageView) {
+    this.waiterImageView = waiterImageView;
+  }
+
+  public ImageView getWaiterImageView() {
+    return waiterImageView;
+  }
+
+  public void moveX(double stepX) {
+    waiterImageView.setLayoutX(waiterImageView.getLayoutX() + stepX);
+  }
+
+  public void moveY(double stepY) {
+    waiterImageView.setLayoutY(waiterImageView.getLayoutY() + stepY);
+  }
+
+  public void move(double stepX, double stepY) {
+    moveX(stepX);
+    moveY(stepY);
+  }
+
+  public void revertMove(double stepX, double stepY) {
+    move(-stepX, -stepY);
+  }
+
+  public void pointUp(String productInHand) throws FileNotFoundException {
```

Comment:
You could combine these methods for less boilerplate code.

Reply:
A great suggestion!